src/realm/alloc_slab.cpp

```diff
@@ -1206,7 +1206,7 @@ void SlabAlloc::extend_fast_mapping_with_slab(char* address)
     // Memory ranges with slab (working memory) can never have arrays that straddle a boundary,
     // so optimize by clamping the lowest possible xover offset to the end of the section.
     new_fast_mapping[m_translation_table_size - 1].lowest_possible_xover_offset = 1ULL << section_shift;
-    m_ref_translation_ptr = new_fast_mapping;
+    m_ref_translation_ptr = new_fast_mapping.release();
```

Comment:
`m_ref_translation_ptr` is an `atomic<RefTranslation*>` so it can't actually be a `unique_ptr`. But al least we are now protected in case anything above throws.